sdk/node/test/mockHsmKeys.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-env mocha */
+
+const chain = require('../dist/index.js')
+const uuid = require('uuid')
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised')
+
+chai.use(chaiAsPromised)
+const expect = chai.expect
+
+const client = new chain.Client()
+
+describe('MockHSM key', () => {
+
+  it('succesfully creates key', () => {
+    return client.mockHsm.keys.create()
+      .then((resp) => expect(resp).not.to.be.empty)
+  })
+
+  it('rejects key creation due to duplicate alias', () => {
+    return client.mockHsm.keys.create({ alias: `key-${uuid.v4()}`})
```

Comment:
Super nit: there is a space after the opening bracket in the params object, but no space before the closing bracket.